tools/content/fetch_messages.dart

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env dart
```

Comment:
Probably cleanest to leave the shebang line (and the executable flag) off of this file, too. Then it's clear there's one intended way to invoke this functionality, via the `check-features` wrapper script.